pkg/controllers/chart/chart.go

```diff
@@ -50,16 +50,18 @@ const (
 	FailurePolicyAbort     = "abort"
 )
 
+// DefaultJobImage now is set in main.go as default value for HelmController.JobImage
 var (
 	commaRE              = regexp.MustCompile(`\\*,`)
 	deletePolicy         = metav1.DeletePropagationForeground
-	DefaultJobImage      = "rancher/klipper-helm:v0.8.2-build20230815"
```

Comment:
This needs to remain an exported variable; we can't just wholesale move it into the tag on the JobImage field, as we need to be able to introspect it from projects that embed this controller.

Ref: https://github.com/k3s-io/k3s/blob/3f237230350b5170eef4e54c7826d88433182efc/pkg/server/server.go#L204-L209

Reply:
@brandond OK, I'm revert DefaultJobImage and I took a different approach with this setting own default image for jobs.